rts/lra/lra-proxy/test/pom.xml

```diff
@@ -26,8 +23,8 @@
         <lra.http.port>8080</lra.http.port>
         <service.http.port>${swarm.http.port}</service.http.port>
 
-        <swarm.debug.params></swarm.debug.params>
-        <swarm.logging.params></swarm.logging.params>
+        <swarm.debug.params/>
```

Comment:
They're just names and besides the name change could brake any existing scripts/notes that use the property. Why do you feel it's important?

Reply:
Because we should aim to provide clear view for any external reader or contributor (and for us in the future) what project do we used. It's a good practice to name with descriptive names. Using descriptive name makes the code/script/configuration file being easier to maintain and less error-prone.
In summary it's not a good practice to refer to an old, deprecated project which is not used in the code base.
It's the reason for doing refactoring. Makes things clearer. We can break some script/notes but as the point of the whole of this jira is movement to thorntail it's logic the swarm is not what is about to be used.

Reply:
I agree with both opinions. I do think that we should look at these properties somewhat like an API though as people may be overriding some of these properties in their own scripts. That said I am now of the opinion that the version number property can safely be changed from version.wildfly-swarm to version.thorntail because the rest of the GAV has changed. If the swarm.debug.params are equivalent between the two then it seems more like we want to treat with deprecation approach, if they have incompatible APIs for the input then I agree that during the change of the dependency it should be updated (either now or as a follow up action)

Reply:
My latest push replaces all occurrences of swarm with thorntail with the justification that the source is implementing a spec that is still at version 1.0-SNAPSHOT and therefore liable to change in arbitrary ways at any time.